core/src/main/java/org/n52/iceland/request/AbstractServiceRequest.java

```diff
@@ -175,6 +180,18 @@ public Locale getRequestedLocale() {
         return LocaleHelper.fromString(getRequestedLanguage());
     }
 
+    public String getOriginalRequest() {
```

Comment:
On second thought, there also is ``isSetRequestedLanguage`` in the same class and the same practice (an ``isSet...`` method in combination with a getter) is used quite often in iceland. Why not change that everywhere?